whylog/constraints/__init__.py

```diff
@@ -164,7 +174,7 @@ def verify(cls, group_contents, param_dict=None):
         - verify({}, ['comp1', 'hello', 'comp1']) returns False
         """
         if len(group_contents) < 2:
```

Comment:
why do we check it on critical path? As I said before, this should be verified once when creating a constraint, not for every single line whylog parses.